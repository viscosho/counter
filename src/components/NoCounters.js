import React from 'react';

const NoCounters = () => {
	return (
		<div className="no-counters p-3 d-flex flex-column justify-content-center align-content-center flex-wrap">
			<h2 className="text-center">No counters yet</h2>
			<p className="text-center">“When I started counting my blessings, my whole life turned around.” —Willie Nelson</p>
		</div>
	);
};

export default NoCounters;
