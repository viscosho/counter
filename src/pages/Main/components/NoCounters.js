import React from 'react';

const NoCounters = () => {
	return (
		<div>
			<h2>No counters yet</h2>
			<p>“When I started counting my blessings, my whole life turned around.” —Willie Nelson</p>
		</div>
	);
};

export default NoCounters;
