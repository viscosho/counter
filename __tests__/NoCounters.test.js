import React from 'react';
import renderer from 'react-test-renderer';
import NoCounters from '../src/components/NoCounters';

it('renders correctly', () => {
	const tree = renderer.create(<NoCounters />).toJSON();
	expect(tree).toMatchSnapshot();
});
