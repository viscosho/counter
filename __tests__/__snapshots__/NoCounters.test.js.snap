// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="no-counters p-3 d-flex flex-column justify-content-center align-content-center flex-wrap"
>
  <h2
    className="text-center"
  >
    No counters yet
  </h2>
  <p
    className="text-center"
  >
    “When I started counting my blessings, my whole life turned around.” —Willie Nelson
  </p>
</div>
`;
